terraform/search.gov.tf

```diff
@@ -0,0 +1,92 @@
+resource "aws_route53_zone" "search_toplevel" {
+  name = "search.gov"
+
+  tags {
+    Project = "dns"
+  }
+}
+
+resource "aws_route53_record" "search_gov_apex" {
+  zone_id = "${aws_route53_zone.search_toplevel.zone_id}"
+  name = "search.gov."
+  type = "A"
+
+  alias {
+    name = "dcp2c9fh8vtdl.cloudfront.net."
+    zone_id = "${local.cloudfront_zone_id}"
+    evaluate_target_health = false
+  }
+}
+
+# www.search.gov — redirects to search.gov through pages_redirect
+resource "aws_route53_record" "search_gov_www" {
+  zone_id = "${aws_route53_zone.search_toplevel.zone_id}"
+  name = "www.search.gov."
+  type = "A"
+
+  alias {
+    name = "dv0x4a4ilr842.cloudfront.net."
+    zone_id = "${local.cloudfront_zone_id}"
+    evaluate_target_health = false
+  }
+}
+
+# find.search.gov
+resource "aws_route53_record" "search_gov_find" {
+  zone_id = "${aws_route53_zone.search_toplevel.zone_id}"
+  name = "find.search.gov."
+  type = "A"
+
+  alias {
+    name = "proxy-east-lb-234082388.us-east-1.elb.amazonaws.com."
+    zone_id = "${local.cloudfront_zone_id}"
```

Comment:
@lsamuels-fearless This should be the ELB zone rather than a CloudFront one. Getting [an error](https://app.circleci.com/pipelines/github/18F/dns/25/workflows/2b9abbad-4d69-41f0-b889-42f01027ce67/jobs/1053) when trying to deploy:

> Tried to create an alias that targets proxy-east-lb-234082388.us-east-1.elb.amazonaws.com., type A in zone Z2FDTNDATAQYW2, but the alias target name does not lie within the target zone

Mind sending another pull request to fix?

Reply:
Ideally this redirects to our search pages, not the Federalist site. I think a CNAME might actually be the answer here. I'll investigate on my side and submit a new PR.

Reply:
Submitted https://github.com/18F/dns/pull/432 to address the above.